management-commands/create-muterole.md

```diff
@@ -1,6 +1,17 @@
 # create-muterole
 
-<TODO: describe>
+This command sets up a role to be used as the mute role for the server. It will edit the permissions of all channels to
+deny the role from sending messages.
+
+It will edit the following permissions form channels and set them to `false` for the role:
```

Comment:
```suggestion
It will edit the following permissions from channels and set them to `false` for the role:
```

also missing the part where the bot also needs all permissions in that channel that it tries to allow or deny in the overwrites